windowing_lib_test.py

```diff
@@ -70,3 +71,59 @@ def test_find_precise_start_time():
     actual_start_time = wl.find_precise_start_time(
         np_front_window_lift, rows, window_size, stride, threshold)
     assert expected_start_time == actual_start_time
+
+
+FRONT_WINDOW_LIFT = [[[1, 1, 2, 2, 2, 2, 20, 20, 20, 20], [1, 1, 2, 2, 2, 2, 20, 20, 20, 20], [
+    1, 1, 2, 2, 2, 2, 20, 20, 20, 20], [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]]]
+NP_FRONT_WINDOW_LIFT = np.array(FRONT_WINDOW_LIFT)
+
+
+def test_get_start_time():
+    expected_start = 6
+    expected_end = 10
+    rows = 4
+    window_size = 2
+    stride = 1
+    variance_threshold = 6
+    actual_start, actual_end = wl.get_start_time(
+        NP_FRONT_WINDOW_LIFT, rows, window_size, stride, variance_threshold)
+    assert expected_start == actual_start
+    assert expected_end == actual_end
+
+
+def test_centered_window1():
+    expected_window = np.array(
+        [[2, 2, 2, 2, 20], [2, 2, 2, 2, 20], [2, 2, 2, 2, 20]])
+    actual_window = wl.centered_window(
+        NP_FRONT_WINDOW_LIFT[0], 5, 5)
+    print(actual_window)
```

Comment:
Do you need these print statements in the test? I'm assuming if there is an error it will print the windows anyways?

Reply:
Nope thanks for the catch just left in after debugging and is by accident.